bundles/CoreBundle/Resources/translations/it.json

```diff
@@ -940,4 +940,4 @@
     "test_url": "Test URL",
     "create_redirects": "Create redirects to keep URLs working (incl. children)?",
     "auto_create": "Auto create"
-}
```

Comment:
please remove entirely if possible

Reply:
I'm trying to revert this, but the blank line still remain